src/os/bluestore/BlueStore.cc

```diff
@@ -4057,7 +4057,6 @@ int BlueStore::_open_db(bool create)
 	  bluefs->get_block_device_size(BlueFS::BDEV_DB) - BLUEFS_START);
       }
       bluefs_shared_bdev = BlueFS::BDEV_SLOW;
-      bluefs_single_shared_device = false;
```

Comment:
This isn't sufficient because bluefs isn't guaranteed to do the wal write on the wal device.  If the device is small and fills up it'll end up on another device.